### DISCUSSION TITLE
Working with query data

### DISCUSSION THREAD

**Discussion by Azique** [Budibase/budibase#1859] [Ideas]
Hey there,

i could not figure out how to use data returned by a query.

say, i have a rest api which returns something like this:

`{
  "videos": [{}]
  "count": 123
}`

i can't figure out how do i iterate through "videos" array and get each videos' title, for example.

by the way, are transformations available somewhere?
like in Tooljet transformation is kind of a part of the query itself. I did see javascript action in Automations, but I don't think there's a way to pass query results there.

**Comment by mjashanks** [comment]
Hi @Azique 

Yes! We need transformers for this. We do not have them currently, but are hoping to get them done in August.

In the meantime, you might be able to accomplish this using the `pluck` handlebars function:

![image](https://user-images.githubusercontent.com/3524181/123601935-23f03600-d7f0-11eb-9ccf-bbac6181fadc.png)

**Reply by Azique** [reply]
hi @mjashanks 

thank you so much!
